Jenkinsfile

```diff
@@ -0,0 +1,41 @@
+def fail(reason) {
```

Comment:
Does the corresponding Jenkins job exist already?

Reply:
It should now